configs/polarRegions.conf

```diff
@@ -51,6 +50,16 @@ regionGroups = ['Antarctic Regions', 'Ocean Basins']
 zmin = -1000
 zmax = 0
 
+[TSDiagramsForOceanBasins]
+## options related to plotting T/S diagrams of major ocean basins
+
+# list of regions to plot or ['all'] for all regions in the masks file.
+# See "regionNames" in the antarcticRegions masks file in
+# regionMaskSubdirectory for details.
+regionNames = ["Atlantic_Basin", "Pacific_Basin", "Indian_Basin",
+               "Arctic_Basin", "Southern_Ocean_Basin",
+               "Global Ocean", "Global Ocean 65N to 65S"]
```

Comment:
These are the same regions we use for the vertical profiles, so it makes sense to restrict the T-S diagrams to this set.